### PR TITLE
Add `rna-transcription` exercise

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

File: exercises/practice/rna-transcription/.meta/example.8th [hunk 1]
```diff
@@ -0,0 +1,5 @@
+needs string/translate
+
+: >rna \ s -- s
+  "GCTA" "CGAU" s:translate
```
Comment (ErikSchierboom) [inline exercises/practice/rna-transcription/.meta/example.8th:4]: There appears to be a bug with `s:tr`: https://8th-dev.com/bin/bugs/tktview/e635f48fe441f2d1a57ff957fb81fe1e83491240

**Review by glennj — APPROVED** [review]

**Comment by ErikSchierboom** [comment]
@axtens ?

**Comment by axtens** [comment]
You knew it was a public holiday here in Australia, didn't you

**Comment by ErikSchierboom** [comment]
> You knew it was a public holiday here in Australia, didn't you

Nope, sorry!

**Comment by axtens** [comment]
Not complaining at all. I'm actually more available than usual. It's ANZAC Day. Google it. It's an interesting story.

**Pull request merged**